.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,10 +15,24 @@ is merged.
 See: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
 -->
 
-
 #### What does this implement/fix? Explain your changes.
 
 
+#### Checklist
+
+<details>
+  <summary>
+   Code change checklist...
+  </summary>
+ 
+ - [ ] Have you updated the relevant docstring/documentation pages?
+ - [ ] Does your change have automated test coverage for the following schema options:
+   - [ ] Dynamic schema
+   - [ ] Sparse data
+ - [ ] Do all exceptions introduced raise appropriate [error messages](https://docs.arcticdb.io/error_messages/)?
+ - [ ] Are API changes highlighted in the above section?
```

Comment:
```suggestion
 - [ ] Are API changes highlighted in the above section?
 - [ ] Is this contribution tested against [all ArcticDB's features](docs/mkdocs/docs/technical/contributing.md)?
```

Reply:
I could remove the dynamic schema/sparse data bit with that?

My problem is that's a long page. And realistically people won't pay attention to every line on that to check a box - I think it's too detailed to be of value in a PR checklist in other words. 

Equally my list is incomplete, so is of limited value in that direction. 

I'm not sure what's most useful in practise.

Reply:
Prefer Julien's suggestion

Reply:
I think that if the review process is properly done, we should not be too concerned with the last item that I proposed. What do you think?

Reply:
I've changed to be a link the @alexowens90 page.